static/js/components/dashboard/FinancialAidCard.js

```diff
@@ -199,9 +199,10 @@ export default class FinancialAidCard extends React.Component {
 
           <Grid className="financial-aid-box">
             <Cell col={12}>
-                Before you can pay, you need to verify your income. Please mail
-                or fax an English-translated and notarized income tax or income
-                statement document. DO NOT SEND BY EMAIL.
+                Before you can pay, you need to verify your income. Please visit
+                the secure website to upload an English-translated and notarized
```

Comment:
"secure website" should be linked to the docusign site.

Reply:
updated.